crates/kitsune_p2p/bootstrap/src/main.rs

```diff
@@ -1,10 +1,32 @@
-#[tokio::main(flavor = "multi_thread")]
+use std::net::IpAddr;
+
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "kitsune-bootstrap",
```

Comment:
I'm trying to keep the p2p in there... to distinguish from other things named kitsune.

```suggestion
    name = "kitsune-p2p-bootstrap",
```